lib/cisco_node_utils/vlan.rb

```diff
@@ -206,5 +224,86 @@ def mapped_vni=(vni)
     def default_mapped_vni
       config_get_default('vlan', 'mapped_vni')
     end
+
+    # Helper method to delete @set_args hash keys
+    def set_args_keys_default
+      @set_args = {}
+    end
+
+    def private_vlan_type
+      result = config_get('vlan', 'private_vlan_type_all')
```

Comment:
We should be able to simplify this changing the yaml entry and then calling config_get with the vlan id.

- `get_value: '/^%d\s+(\S+)$/'`  <--- Add `%d` which gets replaced with the vlan_id
- `config_get('vlan', 'private_vlan_type', @vlan_id)` <-- Call `config_get` with @vlan_id